web-app/src/main/webapp/script/script.js

```diff
@@ -29,7 +39,7 @@ window.onload = async () => {
   document.getElementById('close-menu').addEventListener('click',
     () => document.getElementById('menu').style.display = 'none');
   fetchMarkers(map);
-	const timeFrameOptions = document.getElementById("time-frame-options");
+  const timeFrameOptions = document.getElementById("time-frame-options");
```

Comment:
nit: how come the autoformatter didn't catch this before? I think we should avoid unrelated style fixing in the future.